### DISCUSSION TITLE
How to contribute to the project?

### DISCUSSION THREAD

**Discussion by vax-r** [TwiN/gatus#527] [Q&A] [answered]
Hello I'm a new user of this gatus project. 
I find it very interesting and want to participate in the project, too.
I wonder if there's any documents for new contributors like what's the architecture of gatus and what is each part of code doing?
So it can be easier to start working on the project.

**Comment by TwiN** [comment, marked as answer]
The long answer: https://github.com/TwiN/.github/blob/master/.github/CONTRIBUTING.md

The short answer: you can contribute to any issue you want, but I would recommend focusing on issues labeled with `good first issue`.

**Comment by heitorPB** [comment]
Another option is to improve the test suite :)

**Reply by vax-r** [reply]
@heitorPB , I would love to help if you are willing to let me know how to do it

**Reply by heitorPB** [reply]
`$ make test` runs the test suite. It will show the test coverage for each part of Gatus. Some packages have no test files, while others are below 100%. Pick any part and write tests to ensure valid behavior.

**Reply by vax-r** [reply]
No problem, thanks for the info
I'll write some and see if I can do any help.
